README.md

```diff
@@ -1,12 +1,14 @@
 <h1 align="center">NOTFLIX</h1>
-<p align="center">f@#k netflix use notflix a tool which search magnet links and stream it with webtorrent</p>
+<p align="center">STOP DEGENERACY</p>
+
+Removed the emojis from the script as its kinda cringe imo and made it less able to search porn, by only searching movies and not everything in general. It makes it a lot harder to search and probably almost impossible so yeah. The major difference between this fork and the original is that you select the form of media you want (movie, tv, documentary, anime), this is because I just don't how to do it in a better way, where you can view all content simulatanousely on 1337x without accessing the degenerate content then besides the way I decided to implement it
```

Comment:
Why do emojis hurt you? I think you should re-think what's contributing to open-source and what's the goal of such software. It's definitely not for making the software less 'cringy'.

Reply:
Well initally I just forked it to make my own version which I would prefer, but I thought removing the ability to search porn might be useful, but I didn't realise it would show all my changes. I would rather just have the changes I made to the actual script be included not the rest of my own little edits for myself